acm/src/main/java/org/literacybridge/acm/Constants.java

```diff
@@ -53,6 +53,7 @@ public class Constants {
   public final static String BATCH_RECORD = "record.log";
   public final static String S3_BUCKET = "acm-logging";
   public final static String FUZZY_THRESHOLD = "FUZZY_THRESHOLD";
+  public final static String VOLUME_STEP = "volume_step";
```

Comment:
@nanabingies change the "volume_step" to "VOLUME_STEP" for consistent naming

Reply:
I did that earlier but Bill suggested I use small letters as that matched the naming convention in the firmware config files.